doc/yabar.1.asciidoc

```diff
@@ -328,6 +328,12 @@ internal-option2: "        "; # icons to indicate quarter, half,
 internal-suffix: "%";
 ----
 
+* *YABAR_SONG* - *Current Song*: Uses 'playerctl' to retrieve the name of the currently played song. Example:
+----
+exec: "YABAR_SONG";
+internal-option1:"spotify";
```

Comment:
Add a space after the colon to make it uniform. Also, comment what exactly `internal-option1` defines.